ffmuc-mesh-vpn-wireguard-vxlan/luasrc/lib/gluon/upgrade/400-mesh-vpn-wireguard

```diff
@@ -15,14 +23,14 @@ uci:delete_all('wireguard', 'wireguard', function(peer)
 	return peer.preserve ~= '1'
 end)
 
+-- TODO: fix get_bool(network.wg_mesh.disabled), currently always true if field is empty
```

Comment:
I get "false" when executing `print(require('simple-uci').cursor():get_bool('network', 'wg_mesh', 'disabled'))` when this does not exist..

Reply:
Yeah, but the `not` in front always makes this true. So in case none of these options is given, it enables mesh_vpn instead of turning it off.

Reply:
I removed the respective lines to avoid confusion. I don't think we need this at all